editor/plugins/node_3d_editor_gizmos.cpp

```diff
@@ -3114,16 +3114,16 @@ void ReflectionProbeGizmoPlugin::commit_handle(const EditorNode3DGizmo *p_gizmo,
 	AABB restore = p_restore;
 
 	if (p_cancel) {
-		probe->set_extents(restore.position);
+		probe->set_size(restore.position);
 		probe->set_origin_offset(restore.size);
 		return;
 	}
 
 	UndoRedo *ur = Node3DEditor::get_singleton()->get_undo_redo();
-	ur->create_action(TTR("Change Probe Extents"));
-	ur->add_do_method(probe, "set_extents", probe->get_extents());
+	ur->create_action(TTR("Change Probe Size"));
+	ur->add_do_method(probe, "set_size", probe->get_size());
 	ur->add_do_method(probe, "set_origin_offset", probe->get_origin_offset());
-	ur->add_undo_method(probe, "set_extents", restore.position);
+	ur->add_undo_method(probe, "set_size", restore.position);
```

Comment:
There's a lot of conversion going on here involving `size`, `restore.position` and `restore.size`, and I'm not fully certain what specifically the values in `restore` are. Do you think it's safe to leave as it is, or does it need changing, e.g. multiplying/dividing by 2?

Reply:
Good question. I am not very familiar with the editor code here. Perhaps @Calinou or @KoBeWi knows more about this particular function.

Reply:
It's ok like that. `restore` is the original value of the property.